processor/elasticinframetricsprocessor/testdata/config.yaml

```diff
@@ -1,8 +1,16 @@
-default:
+elasticinframetrics:
+elasticinframetrics/all:
   add_system_metrics: true
+  add_k8s_metrics: true
 
-with_system_metrics:
+elasticinframetrics/with_system_metrics:
   add_system_metrics: true
+  add_k8s_metrics: false
 
-without_system_metrics:
-  add_system_metrics: false
\ No newline at end of file
+elasticinframetrics/without_system_metrics:
```

Comment:
More appropriate name for this ?
As we are disabling not only system but k8s as well

Reply:
done https://github.com/elastic/opentelemetry-collector-components/pull/14/commits/a8c5b77fa34c74ddb64d1c3d119c2df924f2cad9, also added it to tests - this one was missing for some reason